assets/src/application/services/toaster/utilities/getToastContainerStyle.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { PositionsType } from '../types';
+
+/**
+ * adapted from:
+ * @link https://github.com/bmcmahen/toasted-notes/blob/master/src/Alert/Message.ts
+ */
+const getToastContainerStyle = (position: PositionsType): React.CSSProperties => {
+	const style: React.CSSProperties = {
+		display: 'flex',
+		flexDirection: 'column',
+		alignItems: 'center',
+	};
+
+	if (position.includes('right')) {
+		style.alignItems = 'flex-end';
+	} else if (position.includes('left')) {
+		style.alignItems = 'flex-start';
+	}
+
+	return style;
```

Comment:
`emotion` would come handy here if we want to avoid inline styles

Reply:
https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad

Reply:
good points... this was copied from either toasted-notes or Chakra but I missed what was going on, so good catch 👍

Reply:
I want to play a little bit with `emotion`, can I try and change smth here?